src/tags-input.js

```diff
@@ -116,6 +116,25 @@ tagsInput.directive('tagsInput', function($timeout, $document, tagsInputConfig)
         return SUPPORTED_INPUT_TYPES.indexOf(type) !== -1;
     }
 
+    /**
+     * validate comma key
+     *
+     * Event properties to verify ','
+     * Chrome: keyIdentifier: "U+002C"
+     * FF: key
+     * IE: char|key
+     * Safari: -
+     *
+     * @param e
+     * @returns {boolean}
+     */
+    function validateComma(e) {
+        var originalEvent = e.originalEvent || e;
+        var char = e.key || e.char || originalEvent.keyIdentifier;
+        var unsupported = typeof char === 'undefined';
```

Comment:
This patch also won't work out for Safari because the event objects are completely equal there.